### ISSUE TITLE
I can't see the snapshot I created on the main page

### ISSUE THREAD

**Issue by t298** [linuxmint/timeshift#74]
The system I use is arch+gnome,timeshit:22.06.5.I can find the snapshot I created in Browse,But you can't see it on the page.

**Comment by Godot-Fye** [comment]
Same problem here. Seems to be a problem with the last Arch update?

**Comment by fredcw** [comment]
Maybe the snapshot wasn't completed due to bug #69 ?

**Comment by Godot-Fye** [comment]
> Maybe the snapshot wasn't completed due to bug #69 ?

Yep, Timeshift closing a few seconds (sometimes 1-2 minutes) after starting the backup.
